modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -199,7 +201,8 @@ const DatePicker = ({
 					}
 
 					if (moment(value).isValid()) {
-						onChange(moment(value).format(dateMask.toUpperCase()));
+						moment.locale(locale);
+						onChange(moment(value).format('MM/DD/YYYY'));
```

Comment:
Additionally, can we do something like `moment(value).locale(locale).format('MM/DD/YYYY');`?

By the way, can you please clarify why are you using this hard coded format (MM/DD/YYYY)?

Reply:
@natocesarrego, this was Carol who indicated to do, because this is the way back end needs to receive. Without it the date selected blinks. For example, if I select a date in pt_BR, 15/03/20, at first it shows correctly and then it changes to 03/15/20.

Reply:
Thanks for your clarification, @alinedoleron! 😉